src/screens/MainApp/DiagnoseRequest/styles.js

```diff
@@ -1,18 +1,35 @@
 import { StyleSheet } from 'react-native'
+import { theme } from '~/constants'
+import { moderateScale } from 'react-native-size-matters'
 
 const styles = StyleSheet.create({
   container: {
+    margin: theme.sizes.margin,
     flex: 1,
-    alignItems: 'center',
-    justifyContent: 'center'
-  },
-  descriptionAndSubmitContainer: {
-    flex: 0.5,
-    margin: '15%',
-    width: '90%',
-    flexDirection: 'row',
-    justifyContent: 'space-around',
+    flexDirection: 'column',
     alignItems: 'center'
+  },
+  image: {
+    margin: theme.sizes.margin,
+    width: moderateScale(150),
+    height: moderateScale(150),
+    backgroundColor: theme.colors.gray,
+    borderRadius: moderateScale(10)
+  },
+  title: {
+    ...theme.fonts.h1,
+    marginBottom: theme.sizes.margin
+  },
+  subtitle: {
+    ...theme.fonts.h2,
+    marginTop: theme.sizes.margin
+  },
+  description: {
+    ...theme.fonts.description,
+    textAlign: 'center',
+    marginBottom: theme.sizes.margin
+  },
+  button: {
   }
 })
 
```

Comment:
I may be wrong but I do not see this file being used (`DiagnoseRequest` doesn't import it). I suggest removing it.